src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/mathfuncs.txt

```diff
@@ -290,4 +290,22 @@ false
 false
 
 >>IsNumeric([1,2,3,4])
-false
\ No newline at end of file
+false
+
+>>RandBetween(1, 1)
+1
+
+>>RandBetween(5, 1)
+'minValue' cannot be greater than maxValue. (Parameter 'minValue')
```

Comment:
Instead of throwing, we should check this in RandBetween's impl and return an ErrorValue